WordPress/src/main/java/org/wordpress/android/ui/compose/components/buttons/PrimaryButton.kt

```diff
@@ -16,15 +17,30 @@ import androidx.compose.ui.res.dimensionResource
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import org.wordpress.android.R
+import org.wordpress.android.ui.compose.theme.AppColor
 import org.wordpress.android.ui.compose.theme.AppTheme
 
 @Composable
 fun PrimaryButton(
     text: String,
     onClick: () -> Unit,
+    modifier: Modifier = Modifier,
     isInProgress: Boolean = false,
-    modifier: Modifier = Modifier
+    useDefaultMargins: Boolean = true,
+    buttonSize: ButtonSize = ButtonSize.NORMAL,
 ) {
+    var computedModifier: Modifier = modifier
+
+    if (useDefaultMargins) {
+        computedModifier = computedModifier
+            .padding(top = 20.dp, bottom = 10.dp)
+            .padding(horizontal = dimensionResource(R.dimen.jp_migration_buttons_padding_horizontal))
+    }
+
+    buttonSize.height?.let {
```

Comment:
💡 Would it make sense to extract this behavior so it can be reused (like here and on `SecondaryButton`)?
e.g. (not tested)

```
fun Modifier.setButtonSize(buttonSize: ButtonSize): Modifier =
    buttonSize.height?.let {
        this.defaultMinSize(minHeight = it)
    } ?: this
```

And in case `height` is not null something like:
``` 
fun Modifier.setButtonSize(buttonSize: ButtonSize): Modifier =
        this.defaultMinSize(minHeight = buttonSize.height)
```

Reply:
After changing `height` to be non-null I think this extraction is not needed anymore, since I will just relay on:
```kotlin
computedModifier = computedModifier.defaultMinSize(minHeight = buttonSize.height)
```